Manuals/FDS_Verification_Guide/FDS_Verification_Guide.tex

```diff
@@ -5013,7 +5013,7 @@ \subsection{Mass conservation (\texorpdfstring{\textct{vege\_mass\_conservation}
 
 \begin{figure}[ht]
 \centering
-\includegraphics[width=3.2in]{SCRIPT_FIGURES/vege_mass_conservation}
+\includegraphics[width=0.8\textwidth]{SCRIPT_FIGURES/vege_mass_conservation}
```

Comment:
Tom,
1. Please do not change the formatting.
2. Looks like caption is not correct.